README.md

```diff
@@ -96,43 +133,39 @@ In your application info.plist, add the following properties:
 
 > NOTE: If your UPS server installation uses a `self-signed certificate`, you can find a quick solution on how to enable support on our [troubleshooting page](https://aerogear.org/docs/unifiedpush/aerogear-push-ios/troubleshooting/#_question_failure_to_connect_when_server_uses_a_self_signed_certificate), as well as links for further information on how to properly enable it on your iOS production applications.
 
-## Receiving Remote Notifications
-
-There are no extra hooks for receiving notifications with the AeroGear library. You can use the existing delegate for receiving remote notifications while the application is running, like:
-
-```
-- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo {
-  // extract desired value from the dictionary...
-}
-```
 ### Push analytics
 
 If you are interested in monitoring how a push message relates to the usage of your app, you can use metrics. Those emtrics are displayed in the AeroGear UnifiedPush Server's console.
 
-* Send metrics when app is launched due to push notification
+#### Metrics when app is launched
 
 ```objc
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
-{
+- (BOOL)          application:(UIApplication *)application
+didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     [AGPushAnalytics sendMetricsWhenAppLaunched:launchOptions];
     return YES;
 }
-
 ```
-* Send metrics when the app is brought from background to foreground due to a push notification
+
+#### Metrics when the app is brought from background to foreground
 
 ```objc
-- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult result))completionHandler {    
-    [AGPushAnalytics sendMetricsWhenAppAwoken:application.applicationState userInfo: userInfo];
+- (void)   application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo
+fetchCompletionHandler:(void (^)(UIBackgroundFetchResult result))completionHandler {
+    [AGPushAnalytics sendMetricsWhenAppAwoken:application.applicationState userInfo:userInfo];
 }
 ```
-## AeroGear UnifiedPush Server
-
-For more information checkout our [tutorial](http://aerogear.org/docs/unifiedpush/aerogear-push-ios/).
 
 ## Documentation
 
-For more details about the current release, please consult [our documentation](http://aerogear.org/ios/).
+For more details about that please consult [our documentation](http://aerogear.org/ios/).
+
+## Demo apps
```

Comment:
Demo apps are Swift, I think we should remove them as this is an Objective-C framework